### PR TITLE
New alert styling and copy

### PR DESCRIPTION
**Pull request by znegrin**
This is based on the issue [#ENG-838](https://athenianco.atlassian.net/browse/ENG-838), and it adds new styles and behavior for the `Toastify` alerts and a new copy for the success cases to simplify the messaging. 

It is still missing the new copy for the errors as it had several conditions that I wasn't sure about.

image

image

### CODE REVIEW

File: src/js/App.jsx [hunk 1]
```diff
@@ -63,7 +63,17 @@ ReactDOM.render(<>
       </Router>
     </React.StrictMode>
   </Auth0Provider>
-  <ToastContainer />
+  <ToastContainer
+      position="top-center"
+      autoClose={5000}
+      hideProgressBar
+      closeOnClick
+      pauseOnFocusLoss={false}
```
Comment (dpordomingo) [inline src/js/App.jsx:71]: are we really sure about this? It's really annoying when a message disappears without being able to read or process or take a screenshot. 
Same about `pauseOnHover={false}`.
I strongly recommend keeping both.

File: src/js/App.jsx [hunk 2]
```diff
@@ -63,7 +63,17 @@ ReactDOM.render(<>
       </Router>
     </React.StrictMode>
   </Auth0Provider>
-  <ToastContainer />
+  <ToastContainer
+      position="top-center"
+      autoClose={5000}
```
Comment (dpordomingo) [inline src/js/App.jsx:68]: Imo 5s for a success message can be too much, but let's see.

Comment (dpordomingo) [inline src/js/App.jsx:68]: Imo 5s for a success message is too much.

File: src/js/App.jsx (hunk 1 above)
Comment (dpordomingo) [inline src/js/App.jsx:71]: are we really sure about this? It's really annoying when an error message disappears without being able to read or process or take a screenshot. 
Same about `pauseOnHover={false}`.
I strongly recommend keeping both.

File: src/js/App.jsx (hunk 2 above)
Comment (warenlg) [reply src/js/App.jsx:68]: Yes, let's make it shorter. I'm ok with 1s for the success message

File: src/js/App.jsx (hunk 1 above)
Comment (warenlg) [reply src/js/App.jsx:71]: We can keep both, yes.

**Comment by warenlg** [comment]
I'm against having specific messages with proper values because it yields to 4 lines error or success messages. Let's simplify the error messages, please. If we reach a ForbiddenError, then let's display "Permission denied" for instance.

**Comment by dpordomingo** [comment]
@znegrin If you don't feel comfortable adding the error handling that @warenlg requested by https://github.com/athenianco/athenian-webapp/pull/342#issuecomment-631344722, assign me the issue once this is merged, and once you're ready with your part. :)

**Comment by znegrin** [comment]
@dpordomingo I agree with your suggestions. I just added specific values to the messages on `logger.jsx` but it seems that it is not working. Can you please review it in case I am missing something?

For the specific messages for the errors, I didn't change them as I wasn't sure about the meaning of each case, so if you can please help me with that too I would appreciate it. Thank you!

**Comment by dpordomingo** [comment]
I started working on this.
Will rebase and add error handling :D

**Comment by dpordomingo** [comment]
Error messages are now shorter.

I reproduce in the following screenshots the responses that the API can return (but the `422`, which I could not reproduce)

image

Imo with this error messages the user will be able to know what happened, and will be able to let us know as well.

**Pull request merged**